app/src/main/java/org/mozilla/fenix/tabstray/NavigationInteractor.kt

```diff
@@ -60,8 +60,10 @@ interface NavigationInteractor {
 
     /**
      * Called when clicking the close all tabs button.
+     * @param private indicates if tabs are private
+     * @param cancelPrivateDownloadsAccepted user confirmed private downloads cancellation
```

Comment:
`shouldCancelPrivateDownloads` might roll out of the tongue easier?

Apologizes for the run-by comments.

Reply:
```suggestion
     * @param cancelPrivateDownloadsAccepted Whether or not downloads in a private session should be cancelled.
```
For kDocs, I think we usually capitalize and try to not infer too much about what the user did.